### ISSUE TITLE
Papers on Audio Deepfake Detection

### ISSUE THREAD

**Issue by tamlhp** [tamlhp/deepfake-benchmark#4]
Every Breath You Don't Take: Deepfake Speech Detection Using Breath
https://arxiv.org/abs/2404.15143

**Comment by tamlhp** [comment]
Training-Free Deepfake Voice Recognition by Leveraging Large-Scale Pre-Trained Models
https://arxiv.org/abs/2405.02179

**Comment by tamlhp** [comment]
Enhancing Generalization in Audio Deepfake Detection: A Neural Collapse based Sampling and Training Approach
https://arxiv.org/abs/2404.13008

**Comment by tamlhp** [comment]
Exploring Green AI for Audio Deepfake Detection
https://arxiv.org/abs/2403.14290

**Comment by tamlhp** [comment]
Retrieval-Augmented Audio Deepfake Detection
https://arxiv.org/abs/2404.13892

**Comment by tamlhp** [comment]
Cross-Domain Audio Deepfake Detection: Dataset and Analysis
https://arxiv.org/abs/2404.04904

**Comment by tamlhp** [comment]
CtrSVDD: A Benchmark Dataset and Baseline Analysis for Controlled Singing Voice Deepfake Detection
https://arxiv.org/abs/2406.02438

**Comment by tamlhp** [comment]
Harder or Different? Understanding Generalization of Audio Deepfake Detection
https://arxiv.org/abs/2406.03512

**Comment by tamlhp** [comment]
RawBMamba: End-to-End Bidirectional State Space Model for Audio Deepfake Detection
https://arxiv.org/abs/2406.06086

**Comment by tamlhp** [comment]
One-Class Learning with Adaptive Centroid Shift for Audio Deepfake Detection
https://arxiv.org/abs/2406.16716

**Comment by tamlhp** [comment]
Targeted Augmented Data for Audio Deepfake Detection
https://arxiv.org/abs/2407.07598
Accepted in EUSIPCO 2024

**Comment by tamlhp** [comment]
Contextual Cross-Modal Attention for Audio-Visual Deepfake Detection and Localization
https://arxiv.org/abs/2408.01532

**Comment by tamlhp** [comment]
ADD 2023: Towards Audio Deepfake Detection and Analysis in the Wild
https://arxiv.org/abs/2408.04967

**Comment by tamlhp** [comment]
WavLM model ensemble for audio deepfake detection
https://arxiv.org/abs/2408.07414

**Comment by tamlhp** [comment]
Temporal Variability and Multi-Viewed Self-Supervised Representations to Tackle the ASVspoof5 Deepfake Challenge
https://arxiv.org/abs/2408.06922

**Comment by tamlhp** [comment]
Does Current Deepfake Audio Detection Model Effectively Detect ALM-based Deepfake Audio?
https://arxiv.org/abs/2408.10853

**Comment by tamlhp** [comment]
Easy, Interpretable, Effective: openSMILE for voice deepfake detection
https://arxiv.org/abs/2408.15775

**Comment by tamlhp** [comment]
SVDD 2024: The Inaugural Singing Voice Deepfake Detection Challenge
https://arxiv.org/abs/2408.16132

**Comment by tamlhp** [comment]
Exploring WavLM Back-ends for Speech Spoofing and Deepfake Detection
https://arxiv.org/abs/2409.05032

**Comment by tamlhp** [comment]
I Can Hear You: Selective Robust Training for Deepfake Audio Detection
https://arxiv.org/abs/2411.00121

**Comment by tamlhp** [comment]
Toward Transdisciplinary Approaches to Audio Deepfake Discernment
https://arxiv.org/abs/2411.05969

**Comment by tamlhp** [comment]
Comparative Analysis of ASR Methods for Speech Deepfake Detection
https://arxiv.org/abs/2411.17349

**Comment by tamlhp** [comment]
Reject Threshold Adaptation for Open-Set Model Attribution of Deepfake Audio
https://arxiv.org/abs/2412.01425

**Comment by tamlhp** [comment]
CodecFake-Omni: A Large-Scale Codec-based Deepfake Speech Dataset
https://arxiv.org/abs/2501.08238

**Comment by tamlhp** [comment]
Vision Graph Non-Contrastive Learning for Audio Deepfake Detection with Limited Labels
https://arxiv.org/abs/2501.04942

**Comment by tamlhp** [comment]
Transferable Adversarial Attacks on Audio Deepfake Detection
https://arxiv.org/abs/2501.11902

**Comment by tamlhp** [comment]
What Does an Audio Deepfake Detector Focus on? A Study in the Time Domain
https://arxiv.org/abs/2501.13887

**Comment by tamlhp** [comment]
DIN-CTS: Low-Complexity Depthwise-Inception Neural Network with Contrastive Training Strategy for Deepfake Speech Detection
https://arxiv.org/abs/2502.20225

**Comment by tamlhp** [comment]
Measuring the Robustness of Audio Deepfake Detectors
https://arxiv.org/abs/2503.17577

**Comment by tamlhp** [comment]
MAVOS-DD: Multilingual Audio-Video Open-Set Deepfake Detection Benchmark
https://arxiv.org/abs/2505.11109

**Comment by tamlhp** [comment]
What You Read Isn't What You Hear: Linguistic Sensitivity in Deepfake Speech Detection
https://arxiv.org/abs/2505.17513

**Comment by tamlhp** [comment]
Few-Shot Speech Deepfake Detection Adaptation with Gaussian Processes
https://arxiv.org/abs/2505.23619

**Comment by tamlhp** [comment]
Two Views, One Truth: Spectral and Self-Supervised Features Fusion for Robust Speech Deepfake Detection
https://arxiv.org/abs/2507.20417

**Comment by tamlhp** [comment]
Towards Reliable Audio Deepfake Attribution and Model Recognition: A Multi-Level Autoencoder-Based Framework
https://arxiv.org/abs/2508.02521

**Comment by tamlhp** [comment]
Are audio DeepFake detection models polyglots?
https://arxiv.org/abs/2412.17924

**Comment by tamlhp** [comment]
QAMO: Quality-aware Multi-centroid One-class Learning For Speech Deepfake Detection
https://arxiv.org/abs/2509.20679

**Comment by tamlhp** [comment]
Reliable Audio Deepfake Detection in Variable Conditions via Quantum-Kernel SVMs
https://arxiv.org/abs/2512.18797
